modules/avail_creator/src/main/java/org/hawkular/component/avail_creator/Availability.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2015 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.component.avail_creator;
```

Comment:
Should 'avail_creator' be a valid package name? The '_' character is what I'm referring to. AFAIK we have not used '_' in package names? _If this is not true, then disregard my comment_ Otherwise, I don't see why we need to introduce another convention for package names. Because now I have to ask which convention should I use for my package names?